master/internal/db/postgres.go

```diff
@@ -988,6 +1004,25 @@ WHERE trials.experiment_id = $1
 	return numTrials, nil
 }
 
+// ExperimentTrialIDs returns the trial IDs for the experiment.
+func (db *PgDB) ExperimentTrialIDs(expID int) ([]int, error) {
+	var trialIDRows []struct {
```

Comment:
apparently lib/pq has a convenient for this to `Scan(pq.Array(...))` but I didn't see an equivalent in pgx (but I didn't look too hard tbh)